docs/development/how_to_add_a_route.md

```diff
@@ -0,0 +1,175 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+The goal of this document is to cover all the necessary steps to add an API route to Parsec.
+
+In summary:
+1. Add the schema describing the new API route
+2. Generate binding code and serialization tests
+3. Implement server-side code (In-memory & PostgreSQL) and tests
+4. Implement client-side code and tests
+
+> Note that you will need a working development environment. See (./README.md).
+## Schema generation
+
+Let's add a new `dummy` route to `invited_cmds`.
+
+> API routes are also called "commands" in Parsec. There are three command families: `anonymous_cmds` (authentication not required), `authenticated_cmds` (authentication required) and `invited_cmds` (invitation required).
```

Comment:
I think you can move this comment at the beginning of the file (hence this would explain the naming of the file) and also add a line about the unrelated administration REST API:

```suggestion
> Client-to-server RPC API routes are also called "commands" in Parsec. There are three command families: `anonymous_cmds` (authentication not required), `authenticated_cmds` (authentication required) and `invited_cmds` (invitation required).
>
> Note there is another unrelated API for administration purpose that uses REST instead of RPC (as it is expected to be used by admin using curl), this documentation doesn't cover it.
```